definitions/ramda_v0.21.x/flow_>=v0.23.x/ramda_v0.21.x.js

```diff
@@ -0,0 +1,603 @@
+/* @flow */
+/* eslint-disable no-unused-vars, no-redeclare */
+
+type UnaryFn<A,R> = (a: A) => R
```

Comment:
Things outside a `declare module` get declared in the global scope -- so we should try to namespace these appropriately. Alternatively, if it's not really useful to reference these types outside of the module definition itself, you can move them inside the `declare module` and be sure the `declare class`es will be hidden from the global scope (since you are using `declare var exports: ...`).